changelog/0.34.md

```diff
@@ -0,0 +1,98 @@
+- [#1790](https://github.com/elastisys/compliantkubernetes-apps/pull/1790) - apps wc: add networkpolicies for hnc @lunkan93
+- [#1791](https://github.com/elastisys/compliantkubernetes-apps/pull/1791) - config: more templating for gatekeeper @anders-elastisys
+- [#1800](https://github.com/elastisys/compliantkubernetes-apps/pull/1800) - apps: add consolidation of errors in node-local-dns @Elias-elastisys
+- [#1809](https://github.com/elastisys/compliantkubernetes-apps/pull/1809) - apps: add rook-ceph-crashcollector netpols @lunkan93
+- [#1810](https://github.com/elastisys/compliantkubernetes-apps/pull/1810) - config: add falco tty templating @anders-elastisys
+- [#1811](https://github.com/elastisys/compliantkubernetes-apps/pull/1811) - Added support for Flux with self-managed CRDs @AlbinB97
+- [#1813](https://github.com/elastisys/compliantkubernetes-apps/pull/1813) - config: add templating for enabling wc to sc probes @anders-elastisys
+- [#1817](https://github.com/elastisys/compliantkubernetes-apps/pull/1817) - apps: Allow calico-accountant drop and execute in Falco @aarnq
+- [#1819](https://github.com/elastisys/compliantkubernetes-apps/pull/1819) - apps sc: Improved alertmanager custom template @Elias-elastisys
+- [#1823](https://github.com/elastisys/compliantkubernetes-apps/pull/1823) - End-to-end tests for Harbor API @aarnq
+- [#1825](https://github.com/elastisys/compliantkubernetes-apps/pull/1825) - apps sc: Log management set successfulJobsHistoryLimit to 1 @crssnd
+- [#1829](https://github.com/elastisys/compliantkubernetes-apps/pull/1829) - apps: upgrade ingress-nginx controller to 1.8.4 and chart to 4.7.3 @crssnd
+- [#1843](https://github.com/elastisys/compliantkubernetes-apps/pull/1843) - apps: upgrade falco chart and rework exceptions @Eliastisys
+- [#1844](https://github.com/elastisys/compliantkubernetes-apps/pull/1844) - apps wc: Unmanaged user-alertmanager secret @aarnq
+- [#1848](https://github.com/elastisys/compliantkubernetes-apps/pull/1848) - Tweak disk usage alerts @Zash
+- [#1852](https://github.com/elastisys/compliantkubernetes-apps/pull/1852) - increase helm timeout @OlleLarsson
+  - Increased the default helm timeout to 30 mintues
+- [#1853](https://github.com/elastisys/compliantkubernetes-apps/pull/1853) - Refactor update-ips script @simonklb
+- [#1858](https://github.com/elastisys/compliantkubernetes-apps/pull/1858) - Update default upcloud storage class to "standard" @viktor-f
+- [#1861](https://github.com/elastisys/compliantkubernetes-apps/pull/1861) - Add more user-alertmanager config options @viktor-f
+- [#1866](https://github.com/elastisys/compliantkubernetes-apps/pull/1866) - apps: Increase number of falco exceptions @LundqvistAndre
+- [#1874](https://github.com/elastisys/compliantkubernetes-apps/pull/1874) - apps: config option to set hostAliases for blackbox exporter @Pavan-Gunda
+
```

Comment:
Missing entry for trivy-operator upgrade.

Reply:
Done !!